src/components/library/LibraryMangaGrid.tsx

```diff
@@ -11,8 +11,9 @@ import MangaGrid from 'components/MangaGrid';
 import { useLibraryOptionsContext } from 'components/context/LibraryOptionsContext';
 import { StringParam, useQueryParam } from 'use-query-params';
 import { useMediaQuery, useTheme } from '@mui/material';
+import { useTranslation } from 'react-i18next';
 
-const FILTERED_OUT_MESSAGE = 'There are no Manga matching this filter';
+const FILTERED_OUT_MESSAGE = 'LibraryMangaGrid.filter.label.no_matches';
```

Comment:
t('LibraryMangaGrid.filter.label.no_matches')

import t from i18next directly

Reply:
that only works in case the langauge is fixed and can't be changed, otherwise, in case another language gets selected after the first load, the language would not get updated

and atleast I don't want to have to use my native language, which gets selected by default